signingscript/test/__init__.py

```diff
@@ -1,9 +1,42 @@
+import asyncio
 import os
+import pytest
+import tempfile
 
 
 def read_file(path):
-    with open(path) as f:
-        return f.read()
+    with open(path, 'r') as fh:
```

Comment:
I think I deleted the function and re-added, and included the 'r' on the re-add.